agent/engine/serviceconnect/manager_linux.go

```diff
@@ -149,6 +154,20 @@ func (m *manager) augmentAgentContainer(task *apitask.Task, container *apicontai
 	return nil
 }
 
+func getSupportedIPFamilies(task *apitask.Task) string {
+	if task.IsNetworkModeBridge() {
+		return supportIPv4Only
```

Comment:
For Bridge mode can we not set SupportedIPFamilies flag? I think in bridge mode EMS will not receive supportedIPFamilies flag, EMS will remain it is for now. This will save some time on this SupportedIPFamilies computation on EMS.